android/cpp/backends/external.h

```diff
@@ -31,7 +31,7 @@ struct BackendFunctions {
   BackendFunctions(const std::string& lib_path);
 
   ~BackendFunctions() {
-    if (handle) {
+    if (false) {
```

Comment:
This doesn't look right to me.

Reply:
@anhappdev I am basically stopping tflite::SharedLibrary::UnLoadLibrary(handle); or dlclose(handle); call here as a workaround of the app crash.

Reply:
1. Yes, `if (false)` means the code in the if block will never be called, why not just remove it?
2. Does this change have an impact on other backends? I am not familiar with the backend code yet, but the code was there for a reason. So other engineers need to review this.